packages/EasyApiPlatform/src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyApiPlatform\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('easy_api_platform');
```

Comment:
I'm not sure if Symfony will allow an empty tree for configuration, did we try that?

Reply:
I have tried to use this package locally and I had no problems.

Reply:
ok thanks